### DISCUSSION TITLE
Question regarding new achievements

### DISCUSSION THREAD

**Discussion by ghost** [community/community#18467] [Discussions] [answered]
I understand that there has been a lot of hype regarding the new achievements.

I also know that there is a public repo showcasing all possible achievements - https://github.com/Schweinepriester/github-profile-achievements

I've already followed the instructions and believe I have fulfilled the requirements to get some of the badges, yet have not received them.

Is there a lagging time for the achievements to be updated? Anyone knows :/

**Comment by melvincwng** [comment, marked as answer]
Hi @waikitmun,

Below is a compiled knowledge of what I know, after hours of scouring through discussion forums, and trying to understand how the mechanics work (for some of the achievements at least).

1. YOLO -  Merge a PR directly without review (can add a friend as a reviewer). Repo should be public.
2. Pull Shark - Merge at least 2 PRs into a public repo.
3. Galaxy Brain: 
- Answer at least 2 discussions. 
- Take note, you CANNOT mark your own answers to your own questions. 
- They have to be marked by another user. Discussions should be in a public repository. 
- Also if you mark answers to your own questions, there could be a bug whereby the achievement won't show despite having >=2 answers marked by other users.
- Reference: https://github.com/github-community/community/discussions/18454
4. Also, please provide a couple of hours (or maybe up to a day) for your new achievements to show. If they haven't shown by then, perhaps you did not hit some "hidden" criteria. In most cases, developers seem to get their achievements instantly though.
5. Contact 'Github Support' if you face any issues.

The community can feel free to chime in, and provide their own observations on how the requirements of the new achievements, in case I may have missed some stuff out.

Thanks.

**Reply by im-parsa** [reply]
ghabool daram dash harfet sanade

**Reply by mcilhm** [reply]
nice info

**Reply by azamara** [reply]
Thank you for arranging how to obtain achievements.

**Reply by imran-baitham** [reply]
@melvincwng  it's helpful thanks.

**Reply by d0ra1mon** [reply]
thanks

**Reply by prooheckcp** [reply]
To add to this, it doesn't have to be on a public repo. I've been doing them mainly in private ones

**Reply by RoshdiRaed** [reply]
nice !!

**Reply by eslamalbaik** [reply]
You are lying, man

**Reply by Roshdi2** [reply]
Great

**Reply by andersonkmetiuk** [reply]
thanks

**Reply by Hatemfer** [reply]
thnx

**Reply by evolve-games** [reply]
Thank you for the insight.

**Reply by xaca** [reply]
Thank you! this achievements is great to undestant github universe 🥇

**Comment by number0x0001** [comment]
There is some slight lag I believe.

**Reply by JaylenOzi** [reply]
Yes, I have some lags too.

**Reply by RoshdiRaed** [reply]
truly

**Comment by im-parsa** [comment]
na khodye harfesh doroste khoham omad

**Comment by im-parsa** [comment]
please provide a couple of hours (or maybe up to a day) for your new achievements to show. If they haven't shown by then, perhaps you did not hit some "hidden" criteria. In most cases, developers seem to get their achievements instantly though.

**Reply by JaylenOzi** [reply]
I think pull shark is troubled

**Reply by RoshdiRaed** [reply]
I do not think so

**Comment by lovecoding-git** [comment]
test

**Comment by elmobp** [comment]
Usually when I answer it appears instantly

**Comment by TheKEKO** [comment]
new achievements is great 👍

**Reply by utsavdotdev** [reply]
Yes❕

**Reply by Hatemfer** [reply]
for sure

**Comment by imran-baitham** [comment]
@azamara @fynntang, to get the Galaxy Brain achievement badge, you need to fulfill the following criteria:

Discussions should be in a public repository
Get marked as answers to 2 or more questions
The questions cannot be asked by you. In short, you cannot answer your own questions, those do not count.
In fact, answering your own questions could prevent the achievement from displaying in your profile page.
My advice here is to perhaps scour a few discussion forums, and answer a few questions based on your own knowledge/expertise. Perhaps if someone finds your answer helpful, he/she would mark your answer and thus help you to unlock the badge.
Hope these steps help you in achieving your badge! Good luck :)

**Comment by foysal619** [comment]
To claim the "Galaxy Brain" badge in GitHub, you need to create a GitHub Action using a Docker container. Here are some tips to help you claim the badge:

Understand Docker: Docker is a tool that allows you to create, deploy, and run applications in containers. To claim the "Galaxy Brain" badge, you need to be familiar with Docker and how it works.

Learn about GitHub Actions: GitHub Actions is a tool that allows you to automate your workflows in GitHub. Learn how to create and configure workflows in GitHub Actions.

Choose a project: Choose a project that you are passionate about and that aligns with your interests and skills. It will make the process more enjoyable and engaging.

Search for existing Docker containers: Search for Docker containers that already exist in Docker Hub or other container registries. Using an existing container can save you time and effort.

Create a Dockerfile: If you cannot find an existing Docker container that meets your needs, create a Dockerfile to build a custom container. Make sure your Dockerfile is well documented and follows best practices.

Write your GitHub Action: Use the Docker container in your GitHub Action to automate your workflows. Make sure to follow the best practices for writing GitHub Actions.

Test your GitHub Action: Test your GitHub Action thoroughly to ensure it works as intended. You can use GitHub's workflow visualizer to see how your workflow executes.

Publish your GitHub Action: Publish your GitHub Action to the GitHub Marketplace and share it with the community. Make sure to follow the guidelines for publishing GitHub Actions.

Apply for the badge: Once you have published your GitHub Action, apply for the "Galaxy Brain" badge by filling out the form provided by GitHub.

By following these tips, you can claim the "Galaxy Brain" badge in GitHub and demonstrate your expertise in Docker and GitHub Actions.

**Comment by RoshdiRaed** [comment]
I think writing 2 Answer is enough to do that

**Reply by Roshdi2** [reply]
Great

**Reply by Roshdi2** [reply]
Nice

**Comment by nawam727** [comment]
To earn the "Galaxy Brain" badge on GitHub Discussions, you need to provide an answer that receives a significant number of reactions from the community. GitHub Discussions allows users to react to posts with various emojis, such as 👍, ❤️, 🚀, and others. To increase your chances of getting the "Galaxy Brain" badge, you should aim to provide helpful and insightful answers that resonate with the community.

Here are some tips to increase the likelihood of receiving reactions and earning the "Galaxy Brain" badge:

1. **Be Helpful:** Ensure that your answers are genuinely helpful and contribute to resolving the original question or issue.

2. **Be Detailed:** Provide detailed and comprehensive answers that cover the problem thoroughly. Include code samples, references, and any relevant links if necessary.

3. **Be Respectful:** Always maintain a respectful and friendly tone when interacting with others in the discussion.

4. **Be Timely:** Try to respond to questions as soon as possible. Timely responses are often appreciated and tend to receive more reactions.

5. **Use Formatting:** Use proper formatting, such as code blocks, lists, and headings, to make your answer easy to read and understand.

6. **Engage with the Community:** Participate in discussions actively, offer insights, and show interest in other people's questions and answers.

7. **Quality over Quantity:** Focus on quality rather than quantity. Providing a single well-thought-out answer is more likely to receive reactions than several short or incomplete responses.

Remember that the "Galaxy Brain" badge is awarded based on the reactions your answer receives, so the exact number required may vary depending on GitHub's internal algorithms and criteria.

Participating in GitHub Discussions in a positive and constructive manner not only helps you earn badges but also contributes to building a supportive and thriving community around the project you are involved in.
